identifier/vite.config.js

```diff
@@ -29,6 +30,9 @@ export default defineConfig((env) => {
           lintCommand: 'eslint --max-warnings=0 src',
         },
       }),
+      legacy({
+        targets: ['defaults', 'ie 6-11'],
```

Comment:
This plugin has consequences for the Content-Security-Policy, needs to be investigated what and how this actually can work.